bw2parameters/ecoinvent_units.txt

```diff
@@ -0,0 +1,17 @@
+# default "ton" is interpreted as "short ton" (2000 pounds)
+ton = 1000 kg
+
+# ecoinvent specific stuff
+unit = [unit]
+person = [person] = _ = guest
+night = [time]
+EUR2005 = [currency]
+kilo = 1000
+Becquerel = becquerel
+Sm3 = [Sm3]
+
+# other useful definitions (feel free to add more)
+percent = 1/100 = _ = pct
```

Comment:
Can you add a comment for people in the future on what `_` means here?

Reply:
Done